### ISSUE TITLE
Targeting as the GM display random character icon

### ISSUE THREAD

**Issue by DarKDinDoN** [eadorin/target-enhancements#14]
Hello,

When I login as the GM and I target a character without having any token selected, the portrait shown if one of the pcs.
And it's a different one each time I login,

![image](https://user-images.githubusercontent.com/1687854/85569159-9903bb00-b632-11ea-8f63-5ff8a88697b2.png)
On the capture, what you see if the portrait of the paladin which is also the third token from the left. This token is not assigned to be as the GM and I didn't select it as my default portrait image.

**Comment by eadorin** [comment]
the paladin was likely the last selected token. Please click the new "clear
all targets" button on the left and tell me if the problem persists.

**Comment by DarKDinDoN** [comment]
Yep working after clearing targets with that button. And for all subsequent "targeting" (even after reload). Strange but working :)

**Comment by Zamrod** [comment]
This does happen every time you load the world. Even with no tokens selected, it seems to think you have one selected on load. I think it happens on refresh as well.
